package.json

```diff
@@ -15,6 +15,7 @@
     "react-dom": "^17.0.1",
     "react-scripts": "4.0.1",
     "typescript": "^4.1.2",
+    "validator": "^13.5.2",
```

Comment:
Preparation for URL validation update, it has however no function at all at the moment.